lighty-examples/lighty-controller-springboot-netconf/src/main/java/io/lighty/core/controller/springboot/rest/TopologyRestService.java

```diff
@@ -94,15 +95,17 @@ public ResponseEntity putTopologyOperational(@PathVariable final String topology
             tx.commit().get(TIMEOUT, TimeUnit.SECONDS);
             LOG.info("Topology was stored to datastore: {}", topology);
             return ResponseEntity.ok().build();
-        } catch (InterruptedException | ExecutionException | TimeoutException e) {
+        } catch (ExecutionException | TimeoutException e) {
+            e.printStackTrace();
```

Comment:
do not use  printStackTrace(); this will create vulnerability, logging is sufficient.